### PR TITLE
design for all section

### PR DESCRIPTION
**Pull request by mutesialine**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+       <div> 
+        <div class="footer__input-container">
+            <input type="email" class="footer__input" placeholder="Your Email">
+            <button class="button__btn--primary button__btn--primary--radius">Subscribe</button>
+        </div>
+        
+      
+       
+    </footer>
+</body>
```
Comment (chrisostome01) [inline index.html:150]: I don't why you have so many `</body>` tags and we are supposed to have one of these please fix this and I can get a better view of what you have done

**Review by chrisostome01 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (mutesialine) [reply index.html:150]: those body tags were commented but i have fixed it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                <div class="nav__items">
+                    <a class="nav__item">home</a>
+                    <a class="nav__item" alt="">product</a>
+                    <a class="nav__item" alt="">pricing</a>
+                    <a class="nav__item" alt="">contact</a>
+                </div>
+            </div>
+            <div class="button">
+                <a class="button__linkgreen">login</a>
+                <button class="button__btn--primary">Join us <i class="bi bi-arrow-right-short"></i></button>
```
Comment (ritha-kankindi) [inline index.html]: change this to "button button__btn--primary" . This is an element's modifier and it is  not recommended in BEM to use it as a class alone

File: index.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            </div>
+        </div>
+
+        <div class="hero">
+            <p class="hero__title">welcome</p>
+            <h1 class="hero__introduction">A great place to receive care</h1>
+            <p class="hero__subintroduction">Get your best looking fine now</p>
+        <div class="button" >
+        <button class="button__btn--primary">Join us</button>
```
Comment (ritha-kankindi) [inline index.html]: Here also

File: index.html [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+                <div class="container__paragraph">
+                    <p >Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics </p>
+                </div>
+                <a class="button__linkgreen " href="">Learn more<i class="bi bi-chevron-right"></i></a>
+        
+        </div>
+    </div>
+    <div class="team">
+        <div class="container__text">
+            <p class="container__paragraph--color">team</p>
```
Comment (ritha-kankindi) [inline index.html]: Here also

File: index.html [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+            <h2 class="container__heading">Our Team</h2>
```
Comment (ritha-kankindi) [inline index.html]: These elements are outside of there block. ( container's elements inside team block)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title></title>
+</head>
+<body>
+    <!-- header -->
+    <div class="header">
+        <div class="nav">
+            <div class="nav__menu"><i class="bi bi-list"></i></div>
```
Comment (chrisostome01) [inline index.html:21]: Would use the same icon we have in Figma, you can save them as SVG

File: index.html [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+      
+     <div class="text text--left">
+            <p class="text__title ">team</p>
+            <h2 class="text__introduction">Our Team</h2>
+            <p class="text__subintroduction">Problems trying to resolve the conflict between 
+            the two major realms of Classical physics: Newtonian mechanics </p>
+            </div>
+    <div class="user">  
+        <div class="user__card">
+           <div ><img class="user__image" src="./img/fixed-height sec-.png" alt=""></div>
```
Comment (chrisostome01) [inline index.html]: this div should have class name

File: index.html [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+            <i class="bi bi-instagram"></i>
+            <i class="bi bi-facebook"></i>
+            <i class="bi bi-twitter"></i>
+           </div>
+          </div>
+        </div>
+
+
+         <div class="user__card">
+           <div ><img  class="user__image" src="./img/fixed-height sec-3.png" alt=""></div>
```
Comment (chrisostome01) [inline index.html]: same here

File: index.html [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+    </footer>
+</body>
+
+
+
+
+
+
+
+
```
Comment (chrisostome01) [inline index.html]: You have a missing tag here

File: style.css [hunk 10]
```diff
@@ -0,0 +1,321 @@
...
+.button--hero{
+  padding-top:20px;
+}
+
+.main {
+padding: 160px 150px 108px 150px;
+}
+
+.expert-section {
+  display: flex;;
```
Comment (chrisostome01) [inline style.css]: You can remove the second `;`

File: style.css [hunk 11]
```diff
@@ -0,0 +1,321 @@
...
+.text__subintroduction {
+ color: #737373;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+ letter-spacing: 0.2px;
+ font-style: normal;
+
+}
+.button--hero{
```
Comment (chrisostome01) [inline style.css]: You can move this up to where you worked on buttons for better structure

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @mutesialine, I left some comments about your structure

**Review by rithaven — APPROVED** [review]

**Review by chrisostome01 — APPROVED** [review]

**Review by karemarsy — APPROVED** [review]
looks nice

**Pull request merged**